src/main/java/app/bpartners/geojobs/service/geo/detection/ZoneDetectionJobService.java

```diff
@@ -3,19 +3,28 @@
 import app.bpartners.geojobs.endpoint.event.EventProducer;
 import app.bpartners.geojobs.endpoint.event.gen.DetectionTaskCreated;
 import app.bpartners.geojobs.endpoint.event.gen.ZoneDetectionJobStatusChanged;
+import app.bpartners.geojobs.repository.ZoneDetectionJobRepository;
 import app.bpartners.geojobs.repository.model.geo.detection.DetectionTask;
 import app.bpartners.geojobs.repository.model.geo.detection.ZoneDetectionJob;
+import app.bpartners.geojobs.repository.model.geo.tiling.ZoneTilingJob;
 import app.bpartners.geojobs.service.JobService;
 import java.util.List;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Service;
 
 @Service
 public class ZoneDetectionJobService extends JobService<DetectionTask, ZoneDetectionJob> {
+  private final ZoneDetectionJobRepository zoneDetectionJobRepository;
+  private final DetectionMapper detectionMapper;
 
   public ZoneDetectionJobService(
-      JpaRepository<ZoneDetectionJob, String> repository, EventProducer eventProducer) {
+      JpaRepository<ZoneDetectionJob, String> repository,
+      EventProducer eventProducer,
+      ZoneDetectionJobRepository zoneDetectionJobRepository,
```

Comment:
is this really needed ? considering that JpaRepository<ZoneDetectionJob, String> is normally the same as ZoneDetectionJobRepository